### ISSUE TITLE
Negative Integers on the API

### ISSUE THREAD

**Issue by AdamJHall** [oroinc/platform#760]
Hello, 

We have recently run into an issue where we could not pass negative integers to a data API endpoint. This seems due to this code:

```
#Oro\Bundle\EntityExtendBundle\Resources\config\services.yml:391
oro_entity_extend.validation_loader:
        class: %oro_entity_extend.validation_loader.class%
        public: false
        arguments:
            - "@oro_entity_config.provider.extend"
            - "@oro_entity_config.provider.form"
        calls:
            -
                - addConstraints
                -
                    - integer
                    -
                        - Regex:
                            pattern: '/^[\d+]*$/'
                            message: "This value should contain only numbers."
```

We have created our own workaround, but I was just wondering if this was intentional or not?

Cheers,
Adam

**Comment by jakubderdziak** [comment]
@AdamJHall thank you for your report. Bug reported to platform team.

INTERNAL ID: BAP-15726

**Comment by anyt** [comment]
Fixed in 3.1.3. Thank you for your contribution!
